server/cronJobs/UpdateAccountNotiyJob.js

```diff
@@ -0,0 +1,103 @@
+const CronJob = require("node-cron");
+const UserModel = require("../models/User");
+const Mailer = require("../../services/Mailer");
+const keys = require("../../config/keys");
+
+const UpdateAccountNotiyJob = () => {
+    const scheduledJobFunction = CronJob.schedule("0 0 3 * * *", async () => {
+        console.log("UpdateAccountNotiyJob executed on a schedule!");
+        try {
+            const users = await UserModel.find({
+                $and: [
+                    { accountUpdateMailCount: { $lt: 3 } },
+                    {
+                        $or: [
+                            { isEmailVerified: false },
+                            { isPhoneVerified: false },
+                            { isIdentityVerified: false }
+                        ]
+                    },
+                    {
+                        $or: [
+                            { FirstName: null },
+                            { FirstName: '' }
+                        ]
+                    }
+                ]
+            }).select('email FirstName LastName');
+
+
+            if (users && users.length > 0) {
+                let mailerCount = 0;
+                for (const item of users) {
+                    const mailOptions = {
+                        to: "mohammed.zubayr10@gmail.com",
```

Comment:
Will this only go to one user and not the intended user?

Reply:
I have resolved this one, added that email for testing purpose